addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py

```diff
@@ -309,9 +309,10 @@ def _invoice_constraints_cen_en16931_ubl(self, invoice, vals):
             # Non-SEPA international credit transfer, the Payment account identifier (BT-84) shall be present.
             # note: Payment account identifier is <cac:PayeeFinancialAccount>
             # note: no need to check account_number, because it's a required field for a partner_bank
-            'cen_en16931_payment_account_identifier': self._check_required_fields(
-                invoice, 'partner_bank_id'
-            ) if vals['vals']['payment_means_vals_list'][0]['payment_means_code'] in (30, 58) else None,
+            # Constraint suspended as Payment Means is not a required field for Credit Notes (or for Invoices)
+            # 'cen_en16931_payment_account_identifier': self._check_required_fields(
+            #     invoice, 'partner_bank_id'
+            # ) if vals['vals']['payment_means_vals_list'][0]['payment_means_code'] in (30, 58) else None,
```

Comment:
if you want to remove the constraint, remove it, no need to keep it as a comments :smile: